### ISSUE TITLE
CMake sets -std=gnu11 even if option CGLM_USE_C99 is turned on

### ISSUE THREAD

**Issue by Winter091** [recp/cglm#176]
I'm building my game and use cglm as maths library. I have CMakeLists.txt file which sets settings and adds cglm as subdirectory for build: 
```
set(CGLM_SHARED   OFF CACHE BOOL "" FORCE)
set(CGLM_STATIC    ON CACHE BOOL "" FORCE)
set(CGLM_USE_C99   ON CACHE BOOL "" FORCE)
set(CGLM_USE_TEST OFF CACHE BOOL "" FORCE)
add_subdirectory(deps/cglm)
```

Even though `CGLM_USE_C99` is turned on, `compile_commands.json` shows that cglm files are still being compiled with c11 flag:
```
...
{
  "directory": "D:/Projects/C/Testing/build/Release/deps/cglm",
  "command": "D:\\Programming\\MinGW64\\mingw64\\bin\\gcc.exe -DCGLM_STATIC @CMakeFiles/cglm.dir/includes_C.rsp -O3 -DNDEBUG -Wall -Werror -O3 -std=gnu11 -o CMakeFiles\\cglm.dir\\src\\affine.c.obj -c D:\\Projects\\C\\Testing\\deps\\cglm\\src\\affine.c",
  "file": "D:/Projects/C/Testing/deps/cglm/src/affine.c"
},
...
```
I'm not proficient in CMake. Is it my misunderstanding? 

If these lines are changed from:
```
if(CGLM_USE_C99)
  set(C_STANDARD 99)
endif()
```

to:
```
if(CGLM_USE_C99)
  set(CMAKE_C_STANDARD 99)
endif()
```
Then everything starts working as expected.

**Comment by Winter091** [comment]
Fixed with #177

**Comment by recp** [comment]
@Winter091 thanks, the PR is merged 🎉
